apps/anomalist/anomalist_api.py

```diff
@@ -736,3 +736,75 @@ def _sample_variables(variables: List[gm.Variable], n: int) -> List[gm.Variable]
         new_n=len(sample_ids),
     )
     return [v for v in variables if v.id in sample_ids]
+
+
+def get_anomalies_for_chart_ids(
+    chart_ids: Optional[List[int]] = None, anomaly_types: Optional[Tuple[str, ...]] = ("version_change",)
```

Comment:
@pablorosado I just realized that the correct name for version change anomalies is `upgrade_change`

Reply:
Oh, that's right, I just fixed it in this branch, thanks!